README.md

```diff
@@ -307,6 +307,17 @@ all four button states ("default", "default + hover", "casting", "casting + hove
 folder contains grey, black, and blue versions of the icons.
 
 
+### Events
```

Comment:
It looks like we are missing the [`chromecastRequested` event](https://github.com/silvermine/videojs-chromecast/blob/d6b0276ee1658fe43a81bd93626e0d101781d21b/src/js/components/ChromecastButton.js#L71) in this list. Could you add it with the description: "Triggers when the user has requested Chromecast playback using this plugin's Chromecast button"  ? Thanks!

Reply:
Added 👍